src/hotspot/cpu/x86/vm_version_x86.cpp

```diff
@@ -626,10 +626,27 @@ class VM_Version_StubGenerator: public StubCodeGenerator {
 
 uint64_t VM_Version::CPUFeatures_parse(const char *ccstr, uint64_t &glibc_features) {
   assert(ccstr, "CPUFeatures_parse NULL");
-  glibc_features = 0;
+  glibc_features = _glibc_features;
   if (strcmp(ccstr, "native") == 0) {
+    return _features;
+  }
+  if (strcmp(ccstr, "verify") == 0) {
+    verify_glibc_by_reexec = true;
+#if INCLUDE_CPU_FEATURE_ACTIVE || INCLUDE_LD_SO_LIST_DIAGNOSTICS
+    tty->print_cr("-XX:CPUFeatures=verify has no effect as glibc "
+# if INCLUDE_CPU_FEATURE_ACTIVE
+                  "CPU_FEATURE_ACTIVE"
+# elif INCLUDE_LD_SO_LIST_DIAGNOSTICS
+                  "ld.so --list-diagnostics"
+# endif // INCLUDE_LD_SO_LIST_DIAGNOSTICS
+                  " feature is available");
+#endif // INCLUDE_CPU_FEATURE_ACTIVE || INCLUDE_LD_SO_LIST_DIAGNOSTICS
     return Abstract_VM_Version::features();
   }
+  if (strcmp(ccstr, "ignore") == 0) {
+    return _features;
+  }
```

Comment:
Should not be here `ignore_glibc_not_using = true` ? I cannot see a place where ignore_glibc_not_using is set to true, could you point?